modules/asset_import/assimp.h

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.
+// Copyright (c) 2014-2017 Godot Engine contributors (cf. AUTHORS.md)
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// -- Godot Engine <https://godotengine.org>
+
+#ifndef EDITOR_SCENE_IMPORTER_ASSET_IMPORT_H
+#define EDITOR_SCENE_IMPORTER_ASSET_IMPORT_H
+
+#include "core/bind/core_bind.h"
+#include "core/io/resource_importer.h"
+#include "core/vector.h"
+#include "editor/import/resource_importer_scene.h"
+#include "editor/project_settings_editor.h"
+#include "scene/3d/mesh_instance.h"
+#include "scene/3d/skeleton.h"
+#include "scene/3d/spatial.h"
+#include "scene/animation/animation_player.h"
+#include "scene/resources/animation.h"
+#include "scene/resources/surface_tool.h"
+
+#include "thirdparty/assimp/include/assimp/DefaultLogger.hpp"
+#include "thirdparty/assimp/include/assimp/LogStream.hpp"
+#include "thirdparty/assimp/include/assimp/Logger.hpp"
+#include "thirdparty/assimp/include/assimp/matrix4x4.h"
+#include "thirdparty/assimp/include/assimp/scene.h"
+#include "thirdparty/assimp/include/assimp/types.h"
```

Comment:
The include paths for thirdparty code should not be absolute, as we should be able to unbundle the library and build against a system provided one on Linux.
And since you already define include paths for all folders in the `SCsub` (might also be a bit overkill), it should also not be needed, as `DefaultLogger.hpp` etc. should be in the include path.

Reply:
Will try.